kin-core/build.gradle

```diff
@@ -17,6 +18,18 @@ android {
         consumerProguardFiles 'proguard-rules.pro'
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
+
+    buildTypes {
+        debug {
+            testCoverageEnabled true
+        }
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
```

Comment:
?

Reply:
this a robolectric property recommneded in recent versions, probably not required for my use case, but not hurts to have it